pkgs/development/libraries/asio/generic.nix

```diff
@@ -0,0 +1,25 @@
+{stdenv, fetchurl, boost, openssl
+, version, sha256, ...
```

Comment:
Why do you need triple dots here?

Reply:
Not work without triple dots

nix build -f /home/test/src_nix/nixpkgs_my asio_1_10                                                                                                                     
error: anonymous function at /home/test/src_nix/nixpkgs_my/pkgs/development/libraries/asio/generic.nix:1:1 called with unexpected argument 'callPackage', at /home/test/src_nix/nixpkgs_my/lib/customisation.nix:74:12